### DISCUSSION TITLE
Augmentation du prix de libération de prison par récidives

### DISCUSSION THREAD

**Discussion by Jaarzz** [Crownicles/Crownicles#1802] [Nouvelles idées]
J'ai remarqué que lorsque quelqu'un est emprisonné, il suffit juste de demander à quelqu'un de le libérer et hop, on est tranquille (je sais que c'est l'utilité de oa commande /unlock)

Cependant, je pense que cette altération devrait être plus contraignante pour son aventure.

Je propose d'instaurer des paliers pour le prix de la libération d'un joueur
Par exemple :
1er unlock : 3000 💰 
2e unlock : 4000 💰 
3e unlock : 5000 💰 
Etc...

Ça fera sûrement réfléchir les personnes qui volent constamment et qui savent qu'ils seront libérés rapidement
Cela donnera sûrement plus de crédibilité et d'intérêt à cette altération

**Comment by voltou91** [comment]
Eeeeeh c'est pour m'abattre que tu dit ça ? En vrai j'aurais dit non sauf si chaque semaine la cumulation reviens à 0. Genre récidive dans la semaine ça augmente le prix.
